src/js/types/generic.type.ts

```diff
@@ -14,7 +14,31 @@ export type Menu = GenericObject;
 
 export type Tooltip = GenericObject;
 
-export type Localization = GenericObject;
+export type StringObject = {
+    [key: string]: string;
+};
+
+export type Localization = {
```

Comment:
Don't really have an idea of where to put this (maybe it's own `localization.type.ts` file in `src/types`?), but I thought it might be good to start making a more specific Localization type.

Reply:
Maybe language.js->ts ?